eng/jacoco-test-coverage/pom.xml

```diff
@@ -110,11 +110,11 @@
       <artifactId>azure-analytics-synapse-artifacts</artifactId>
       <version>1.0.0-beta.9</version> <!-- {x-version-update;com.azure:azure-analytics-synapse-artifacts;current} -->
     </dependency>
-    <dependency>
-      <groupId>com.azure</groupId>
-      <artifactId>azure-analytics-synapse-spark</artifactId>
-      <version>1.0.0-beta.6</version> <!-- {x-version-update;com.azure:azure-analytics-synapse-spark;current} -->
-    </dependency>
+<!--    <dependency>-->
+<!--      <groupId>com.azure</groupId>-->
+<!--      <artifactId>azure-analytics-synapse-spark</artifactId>-->
+<!--      <version>1.0.0-beta.5</version> &lt;!&ndash; {x-version-update;com.azure:azure-analytics-synapse-spark;current} &ndash;&gt;-->
+<!--    </dependency>-->
```

Comment:
Was this library causing issues?

Reply:
It had flaky tests that were causing the whole pipeline to fail. I have re-enabled it now to see if it has been fixed.